InternalDocs/locations.md

```diff
@@ -5,20 +5,22 @@ representation of the source code positions of instructions, which are
 returned by the `co_positions()` iterator.
 
 `co_linetable` consists of a sequence of location entries.
-Each entry starts with a byte with the most significant bit set, followed by zero or more bytes with most significant bit unset.
+Each entry starts with a byte with the most significant bit set, followed by
+zero or more bytes with most significant bit unset.
 
 Each entry contains the following information:
-* The number of code units covered by this entry (length)
+
+* The number of code units covered by this entry (length).
```

Comment:
Because I forgot :') thanks for noticing it. I'll just remove the period but will keep the new line before `*` (it's just a bit easier to read when you don't render the markdown file).